spec/unit/filters/filter_form_builder_spec.rb

```diff
@@ -3,7 +3,7 @@
   # Setup an ActionView::Base object which can be used for
   # generating the form for.
   let(:helpers) do
-    view = action_view
+    view = mock_action_view
```

Comment:
I'm confused a little with these changes because, the original method was also called `mock_action_view` or at least that was what was moved and edited. Is `action_view` a method/variable we can remove or its elsewhere? I mention this since I've seen this changed in several other files but nothing changing something called `action_view` so was I was curious.

Reply:
You're right! I missed that! I see it now in the diff. I knew I was missing something. Just wanted to be sure. Thanks! Yes I think what you did is a great improvement. It helps make things a little clearer. 👍🏻